packages/web-player-component/src/index.js

```diff
@@ -0,0 +1,74 @@
+//import { textChangeRangeIsUnchanged } from 'typescript';
+import WebPlayer from '@eyevinn/web-player-core';
+import { renderEyevinnSkin } from '@eyevinn/web-player-eyevinn-skin';
+
+import style from '@eyevinn/web-player-eyevinn-skin/dist/index.css';
+
+export default class PlayerComponent extends HTMLElement {
+  static get observedAttributes() {
+    return ['source', 'starttime', 'muted', 'autoplay'];
+  };
+  constructor() {
+    //Call constructor of HTMLElement
+    super();
+    //Attach CSS
```

Comment:
since you're not using the ShadowDOM anymore it's important to not that the styling will leak into the rest of the DOM, it's most likely fine but something to consider.

Reply:
Why did you choose not to use the ShadowDOM anymore, just of out of curiosity?

Reply:
@slowmove suggested to not use a ShadowDOM.

Reply:
I wrote `We should discuss whether we should use a shadow dom at all vs render in the main document`

Reply:
Let's render in the main document for now?

Reply:
I think we should render it in the shadowDom to encapsualte the CSS. I know there was limited support for that a few years ago but it seems to be supported everywhere now.

Reply:
Added shadow DOM again!